### ISSUE TITLE
Composer

### ISSUE THREAD

**Issue by hfrahmann** [ccampbell/chromephp#22]
Can you add chromePHP to https://packagist.org/ for composer-support?

**Comment by jkrehm** [comment]
As a stop-gap approach until the author creates a composer.json (or one of us does and submits a pull request), I'm using the follow composer.json in my project:

```
{
    "repositories": [
        {
            "type": "package",
            "package": {
                "name": "ccampbell/chromephp",
                "version": "dev-master",
                "dist": {
                    "url": "https://github.com/ccampbell/chromephp/archive/master.zip",
                    "type": "zip"
                },
                "source": {
                    "url": "https://github.com/ccampbell/chromephp.git",
                    "type": "git",
                    "reference": "master"
                },
                "autoload": {
                    "classmap": ["ChromePhp.php"]
                }
            }
        }
    ],
    "require": {
        "ccampbell/chromephp": "dev-master"
    }
}
```

Then I install the project using

```
composer install
```

I pieced this together with the help of [this article](http://www.craftitonline.com/2012/03/how-to-use-composer-without-packagist-like-a-newbie/).

**Comment by pcalves** [comment]
Cheers, @jkrehm! I was thinking of forking the project, creating a composer.json and submitting that to Packagist myself, but I'm not really sure if that's the right way to go about it.

**Comment by jkrehm** [comment]
I thought about doing that myself (even started to) but it'd probably be better to fork it, make the composer.json and then submit a pull request. If that gets ignored for a while, though... I dunno.

**Comment by ccampbell** [comment]
Sorry, I don't really find time to work on this project these days.  I am not familiar with composer.  If someone else wants to submit a pull request then I will be more than happy to merge it.

**Comment by ccampbell** [comment]
This was resolved with #26
